index.html

```diff
@@ -2,236 +2,338 @@
-        <h2 class="proArt">Profesional Art Printing Data</h2>
-        <p class="story2">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard
-        </p>
-        <ul class="grid-container2">
-          <li class="stack2">html</li>
-          <li class="stack2">bootstrap</li>
-          <li class="stack2">Ruby</li>
-        </ul>
-        <a href="#" class="button">See Project</a>
-      </div>
-    </section>
-
-    <section class="item-3">
-      <div class="about">
-        <h2 class="aboutMe">About me</h2>
-        <p class="decription">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <a class="seeProject">Get My Resume</a>
-        <img class="bottomline" src="assets/buttomline.png" alt="border line" />
-        <div class="language">
-          <div class="triangle">
-            <img class="rect" src="assets/Rectangle%2055.png" alt="rentangle" />
-          </div>
-          <ul class="langElement">
-            <li class="lang">Language</li>
-            <li class="javascript">Javascript</li>
-            <li class="ruby">Ruby</li>
-            <li class="html">Html</li>
-            <li class="css">Css</li>
+    <main>
+      <section class="headline">
+        <div class="social">
+          <ul>
+            <li>
+              <a href="#"
+                ><img src="./assets/icons/github.png" alt="github icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/icons/linkedin.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/icons/angelist.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/icons/twitter.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/icons/medium.png" alt="" /></a>
+            </li>
           </ul>
         </div>
-
-        <div class="framework">
-          <div class="dot">
-            <img class="rect" src="assets/square.png" alt="square" />
+        <div class="content">
+          <h1>Hey There.<br />I'm Chidiebere</h1>
+          <h2>I'am a Software Developer</h2>
+          <p>
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, <br />don't hesitate to contact me.
+          </p>
+        </div>
+        <div class="background"></div>
+      </section>
+      <section class="works">
+        <div class="title">
+          <h2>My Recent Works</h2>
+          <div class="separator"></div>
+        </div>
+        <div class="featured">
```

Comment:
- [OPTIONAL] Good job on adding style for work section but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
Here you have a screenshot of your project:

![image](https://user-images.githubusercontent.com/74506933/154802109-5612034c-6581-4a6a-9fec-d920e47e5e69.png)

Here you have a screen of how it should be:

![image](https://user-images.githubusercontent.com/74506933/154802122-e9e34550-06a5-4901-a0eb-76be6a153ca6.png)